src/Symfony/Component/Intl/Data/Bundle/Compiler/GenrbCompiler.php

```diff
@@ -34,13 +34,13 @@ class GenrbCompiler implements BundleCompilerInterface
      */
     public function __construct(string $genrb = 'genrb', string $envVars = '')
     {
-        exec('which '.$genrb, $output, $status);
+        $this->genrb = ($envVars ? $envVars.' ' : '').$genrb;
+
+        exec(escapeshellcmd('which '.$this->genrb), $output, $status);
```

Comment:
escapeshellcmd should wrap only `$genrb` otherwise we're runnint the "which $this->genrb" command, which doesn't exist